samples/minimaloboe/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="purple_200">#FFBB86FC</color>
+    <color name="purple_500">#FF6200EE</color>
+    <color name="purple_700">#FF3700B3</color>
```

Comment:
nit: Seems like the only color defined here that is used is purple_700. Is this ok?

Reply:
This was default code provided by Android Studio.